virt/nodes/virt-about-node-maintenance.adoc

```diff
@@ -6,14 +6,95 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-include::modules/virt-about-node-maintenance.adoc[leveloffset=+1]
+Nodes can be placed into maintenance mode by using the `oc adm` utility or `NodeMaintenance` custom resources (CRs).
 
-include::modules/virt-about-runstrategies-vms.adoc[leveloffset=+1]
+[NOTE]
+====
+The `node-maintenance-operator` (NMO) is no longer shipped with {VirtProductName}. It is deployed as a standalone Operator from the *OperatorHub* in the {product-title} web console or by using the OpenShift CLI (`oc`).
+
+For more information on remediation, fencing, and maintaining nodes, see the link:https://access.redhat.com/documentation/en-us/workload_availability_for_red_hat_openshift/23.2/html-single/remediation_fencing_and_maintenance/index#about-remediation-fencing-maintenance[Workload Availability for Red Hat OpenShift] documentation.
+====
+
+[IMPORTANT]
+====
+Virtual machines (VMs) must have a persistent volume claim (PVC) with a shared `ReadWriteMany` (RWX) access mode to be live migrated.
+====
+
+The Node Maintenance Operator watches for new or deleted `NodeMaintenance` CRs. When a new `NodeMaintenance` CR is detected, no new workloads are scheduled and the node is cordoned off from the rest of the cluster. All pods that can be evicted are evicted from the node. When a `NodeMaintenance` CR is deleted, the node that is referenced in the CR is made available for new workloads.
+
+[NOTE]
+====
+Using a `NodeMaintenance` CR for node maintenance tasks achieves the same results as the `oc adm cordon` and `oc adm drain` commands using standard {product-title} custom resource processing.
+====
+
+[id="eviction-strategies"]
+== Eviction strategies
+
+Placing a node into maintenance marks the node as unschedulable and drains all the VMs and pods from it.
+
+You can configure eviction strategies for virtual machines (VMs) or for the cluster.
+
+VM eviction strategy::
+
+The VM `LiveMigrate` eviction strategy ensures that a virtual machine instance (VMI) is not interrupted if the node is placed into maintenance or drained. VMIs with this eviction strategy will be live migrated to another node.
++
+You can configure eviction strategies for virtual machines (VMs) by using the xref:../../virt/getting_started/virt-web-console-overview.adoc#virtualmachine-details-scheduling_virt-web-console-overview[web console] or the xref:../../virt/live_migration/virt-configuring-live-migration.adoc#virt-configuring-a-live-migration-policy_virt-configuring-live-migration[command line].
```

Comment:
The command line link goes to "Configuring a live migration policy..." and should be changed to link to the eviction strategy doc instead